Mail/Views/Thread/Message/Attachment/AttachmentsView.swift

```diff
@@ -117,8 +117,14 @@ struct AttachmentsView: View {
             try? await mailboxManager.swissTransferAttachment(message: message)
         }
         .sheet(item: $previewedAttachment) { previewedAttachment in
-            AttachmentPreview(attachment: previewedAttachment)
-                .environmentObject(mailboxManager)
+            if #available(iOS 18.0, *) {
+                AttachmentPreview(attachment: previewedAttachment)
+                    .environmentObject(mailboxManager)
+                    .presentationSizing(.page)
+            } else {
+                AttachmentPreview(attachment: previewedAttachment)
+                    .environmentObject(mailboxManager)
+            }
```

Comment:
AttachmentPreview is also displayed on other screens. You can refactor to a view extension that does nothing < iOS 18.
For example: `Toolbar+Extension.swift:33`